Forms/ReadonlyField.php

```diff
@@ -61,6 +61,12 @@ public function castingHelper($field) {
 		return parent::castingHelper($field);
 	}
 
+	public function getSchemaStateDefaults() {
+		$values = parent::getSchemaStateDefaults();
+		$values['value'] = $this->dataValue();
+		return $values;
```

Comment:
Maybe leave a comment along the lines of "so we do not end up with `<i>('none')</i>` in the value.